web/frontend/src/components/slam/index.svelte

```diff
@@ -64,7 +73,8 @@ const refresh2d = async () => {
      * to see if a change has been made to the pointcloud map.
      * A new call to getPointCloudMap is made if an update has occured.
      */
-    if (mapTimestamp?.getSeconds() === lastTimestamp.getSeconds()) {
+
```

Comment:
[nit] Should remove the empty line between the comment and the corresponding code